src/Behat/Context/DefaultCookieManager.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Metadrop\Behat\Context;
+
+/**
+ * Default cookie manager implementation.
+ *
+ * Manages cookie consent interactions.
```

Comment:
Proposed coomment instead of "Manages cookie consent interactions.":

"Manages cookie consent interactions with an unknown cookie manager. This implies that the selector for the banner and reject and accept buttons must be provided. Also, accepting and rejecting cookies can't be done programmatically, because the cookie manager is unknown and there is no way to automatically discover how to do it."